### ISSUE TITLE
mynt 0.2.3 and latest misaka don't mix

### ISSUE THREAD

**Issue by bobcatfish** [pyladies/pyladies#232]
## TL; DR

The instructions on the pyladies repo for how to run locally do not currently work! The latest version of misaka is causing problems.
## The problem

I tried to follow along with [the steps to run the pyladies website locally](https://github.com/pyladies/pyladies#to-run-locally) and I ran into a bit of trouble when running `(PyLadies) $ mynt gen -f _site && mynt serve _site`:

```
...
  File "/Users/christie/pyladies_site/pyladies_env/lib/python2.7/site-packages/mynt/parsers/markdown/misaka.py", line 70, in Parser
    'lax_html_blocks': m.EXT_LAX_HTML_BLOCKS,
AttributeError: 'module' object has no attribute 'EXT_LAX_HTML_BLOCKS'
```

After looking into it a bit, it looks like [those attributes are not exposed in the latest version of misaka](https://github.com/FSX/misaka/blob/master/misaka/api.py#L10), 2.0.0:

```
>>> import misaka as m
>>> m.html(
    'Hello, world!',
    m.EXT_AUTOLINK | m.EXT_TABLES,
    m.HTML_EXPAND_TABS
)
Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
AttributeError: 'module' object has no attribute 'HTML_EXPAND_TABS'
```

So I removed misaka 2.0.0 and installed the previous version, 1.0.2, and I was able to import `EXT_LAX_HTML_BLOCKS`:

```
>>> import misaka as m
>>> m.EXT_LAX_HTML_BLOCKS
32
```

So this is kind of an mynt bug, since mynt is using misaka. However mynt has moved on, [the latest version is 0.3.1](https://github.com/Anomareh/mynt/releases). So I can't submit a fix to them to depend on the earlier misaka.

"Why doesn't this fail on the build agent?" I hear you ask. Looking at output [from a recent build](https://circleci.com/gh/pyladies/pyladies/100) it looks like misaka is already installed on the build agent, so it likely hasn't been updated in a while:

```
Requirement already satisfied (use --upgrade to upgrade): misaka>=1.0.2 in /home/ubuntu/virtualenvs/venv-2.7.6/lib/python2.7/site-packages (from mynt==0.2.3)
```
## Solutions

For fixes, some ideas:
- Easiest: add misaka == 1.0.2 to requirements.txt (and to circle.yml I suppose!)
- Harder: update the version of mynt used; presumably the latest version doesn't have this problem

Thanks!

**Comment by bobcatfish** [comment]
Another possible solution from https://github.com/pyladies/pyladies/issues/161: not using mynt at all :D

**Comment by treyhunner** [comment]
@bobcatfish :+1: thank you for opening this!  While testing out #235 I couldn't figure out what was wrong until I found this issue.  A `pip install misaka == 1.0.2` fixed everything for me! :smile:

**Comment by TechWhizZ199** [comment]
Is this group, restricted just to ladies? :)

**Comment by econchick** [comment]
@TechWhizZ199 on [pyladies.com](http://pyladies.com) it says:

> ...Anyone with an interest in Python is encouraged to participate!

So no, it's not just for women. It's a group meant to help improve diversity of the Python community.

**Comment by totallygloria** [comment]
Thanks so much for the work-around -- I was getting the same errors and unsure where to head. Works for me, too!

**Comment by bobcatfish** [comment]
I've submitted a pull request to fix this by updating to the latest mynt https://github.com/pyladies/pyladies/pull/249

Will leave the exercise of switching to something other than mynt for https://github.com/pyladies/pyladies/issues/161 :)

**Comment by BethanyG** [comment]
@bobcatfish thank you *SO MUCH* - I just ran into this issue as well, and downgrading to misaka 1.0.2 fixed it immediately.  Looks like #249 is *not quite* closed yet, but soon.  :-)

**Comment by willingc** [comment]
Closing this issue since upgrading the infrastructure for this site will likely move away from Mynt.
